src/utils/getPRBlock.js

```diff
@@ -0,0 +1,9 @@
+const { whitespace, beginningOfStringOrNewline } = require('./regexes')
+const { listPrefix, anyLink } = require('./connectedPRRegexes')
+
+const prBlockRegex = `${beginningOfStringOrNewline}(?<block>#* ?\\*{0,2}pull requests:?\\*{0,2}:?(${whitespace}*${listPrefix}${anyLink})*)`
```

Comment:
It seems like if alan (`@Stackblocks` was ever mentioned in a PR then the PR would be blocked.. is that correct? Could the block be done via a label instead?

Reply:
why would it be blocked?